### DISCUSSION TITLE
In the saved TAB group, you can create a new TAB to open the blocked site

### DISCUSSION THREAD

**Discussion by robertdowney21** [proginosko/LeechBlockNG#436] [Q&A]
I usually save group tabs and inadvertently found this problem.

**Comment by proginosko** [comment]
Which browser are you using?

**Reply by robertdowney21** [reply]
Chrome. This save TAB grouping feature seems to be a recent update.

**Comment by proginosko** [comment]
Thanks. See the discussion here: https://github.com/proginosko/LeechBlockNG-chrome/issues/67

**Reply by robertdowney21** [reply]
Thank you very much, but I was hoping for a better solution

**Reply by proginosko** [reply]
Me too! But unfortunately I don't have one yet.
